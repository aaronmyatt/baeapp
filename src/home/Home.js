import React, { Component } from 'react'
import './Home.css'


class Home extends Component {
    constructor (props) {
        super(props)
        this.state = null
    }

  render() {
    return (

        <div>
            < HomeText />
        </div>
    );
  }
}

class HomeText extends Component {
  constructor (props) {
      super(props)
      this.state = null
  }

  render() {
    return (
        <div>
            <div>
                <h2>SPEC</h2>
                <p>Providing location info in JSON format (<a href="https://github.com/aaronmyatt/baeapp/blob/master/location.json">locations</a>). Letâ€™s build a single page application:</p>
            </div>
            <ol>
                <li>Group those location by country and city.</li>
                <li>Show the location with address, city and post with its location in map.</li>
                <li>Provide documentation about technology used and flow.</li>
                <li>Unit test is optional</li>
            </ol>
        </div>
    );
  }
}

export default Home
