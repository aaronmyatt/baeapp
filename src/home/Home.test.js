import React from 'react'
import renderer from 'react-test-renderer'
import Home from './Home'


describe("home page", () => {

    test('renders static content', () => {
      const output = renderer.create(< Home />);
      expect( output.toJSON() ).toMatchSnapshot()
    })

})
