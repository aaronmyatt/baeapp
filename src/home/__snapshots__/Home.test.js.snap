exports[`home page renders static content 1`] = `
<div>
  <div>
    <div>
      <h2>
        SPEC
      </h2>
      <p>
        Providing location info in JSON format (
        <a
          href="https://github.com/aaronmyatt/baeapp/blob/master/location.json">
          locations
        </a>
        ). Letâ€™s build a single page application:
      </p>
    </div>
    <ol>
      <li>
        Group those location by country and city.
      </li>
      <li>
        Show the location with address, city and post with its location in map.
      </li>
      <li>
        Provide documentation about technology used and flow.
      </li>
      <li>
        Unit test is optional
      </li>
    </ol>
  </div>
</div>
`;
